### ISSUE TITLE
Add response output to docx and csv

### ISSUE THREAD

**Issue by bfuzzy1** [preludeorg/operator-support#212]

**Comment by privateducky** [comment]
@bfuzzy1 i tend to leave response data out of reports for two reasons:

1. it adds complexity to the report which tends to confuse the (usually) non-technical consumer of the report.
2. to avoid dumping anything sensitive into an immediately shared PDF/doc report.

That said, we could add an option to include the response data to the export. But I'm curious your thoughts/case first.

**Comment by bfuzzy1** [comment]
Makes sense I was thinking more from the perspective of the operator and being able to dump those responses from specific commands to be able to show evidence of returned values from something like querying domain admins and being able to give stakeholders some evidence of the output of a particular command.

**Comment by bfuzzy1** [comment]
Maybe having it dump that into just csv output would be a better approach vs inside the reporting template?

**Comment by privateducky** [comment]
yep, for sure. it'd be easy enough to add as an option to check before export. just curious your use case ;) 

this seems reasonable enough to get in the next version of the plugin.
